python/py-brotlicffi/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-brotlicffi
+version             1.0.9.2
+license             MIT
+
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+description         Python CFFI bindings to the Brotli library
+long_description    {*}${description}
+
+homepage            https://github.com/python-hyper/brotlicffi
+
+checksums           rmd160  a157bc47475ffa6812cea70ff11ec6701b3325e6 \
+                    sha256  0c248a68129d8fc6a217767406c731e498c3e19a7be05ea0a90c3c86637b7d96 \
+                    size    440077
+
+python.versions     37 38 39 310 311
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-cffi
+
+    set pybrotlipy_name py${python.version}-brotlipy
+} else {
+    set pybrotlipy_name py-brotlipy
+}
+
+#------------------------------------------------------------------------------
+# NOTE: Remove this logic - along with declaration of 'pybrotlipy_name', above,
+#   when obsolete port 'py-pybrotlipy' is ultimately deleted.
+#------------------------------------------------------------------------------
+pre-activate {
+    # Deactivate obsolete port 'py-crypto', if installed
```

Comment:
Should be 'py-brotlipy', right?

Reply:
thanks to catching this copy and paste issue!